drools-wb-screens/drools-wb-guided-dtree-editor/drools-wb-guided-dtree-editor-client/src/main/java/org/drools/workbench/screens/guided/dtree/client/widget/shapes/BaseGuidedDecisionTreeShape.java

```diff
@@ -172,32 +172,27 @@ public void execute() {
     protected Group setupControl( final ImageResource resource,
                                   final Command command ) {
         final Group controlGroup = new Group();
-        final Picture picture = new Picture( resource,
-                                             false );
-        picture.onLoaded( new PictureLoadedHandler() {
-
-            @Override
-            public void onPictureLoaded( Picture picture ) {
-                //This is a hack for Lienzo 1.2 (and possibly 2.x?). There is a bug in Picture when
-                //we want to add it to Lienzo's SelectionLayer. We work around it here by adding
-                //the Picture to a Group containing a "near invisible" Rectangle that we use to
-                //capture the NodeMouseClickEvents.
-                final double offsetX = -picture.getImageData().getWidth() / 2;
-                final double offsetY = -picture.getImageData().getHeight() / 2;
-                final Rectangle selector = new Rectangle( picture.getImageData().getWidth(),
-                                                          picture.getImageData().getHeight() );
-                selector.setFillColor( Color.rgbToBrowserHexColor( 255,
-                                                                   255,
-                                                                   255 ) );
-                selector.setAlpha( 0.01 );
-                selector.setLocation( new Point2D( offsetX,
-                                                   offsetY ) );
-                picture.setLocation( new Point2D( offsetX,
-                                                  offsetY ) );
-                controlGroup.add( picture );
-                controlGroup.add( selector );
-            }
-        } );
+        new Picture( resource, picture1 -> {
```

Comment:
Hi @romartin 

The workbench build now fails with this message:
[ERROR] Errors in 'jar:file:/home/jomarko/.m2/repository/org/drools/drools-wb-guided-dtree-editor-client/7.0.0-SNAPSHOT/drools-wb-guided-dtree-editor-client-7.0.0-SNAPSHOT-sources.jar!/org/drools/workbench/screens/guided/dtree/client/widget/shapes/BaseGuidedDecisionTreeShape.java'
[INFO]          [ERROR] Line 175: The constructor Picture(ImageResource, (<no type> picture1) -> {}, boolean) is undefined
[INFO]          [ERROR] Line 175: The target type of this expression must be a functional interface


Please, could you investigate the problem? Thanks.